tests/integration/cli/client_index_create_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package clitest
+
+import (
+	"testing"
+)
+
+func createUserCollection(t *testing.T, conf DefraNodeConfig) {
+	createCollection(t, conf, `type User { Name: String }`)
```

Comment:
todo: We agreed to follow the convention of `Type` and `field` for our schemas. This should be changes to `type User { name: String }`.